### PR TITLE
Sa-2: Week 8: Created Api's

### PR DESCRIPTION
**Pull request by Mayankingale123**
created an api folder in central mess module which contains three files serializers.py, views.py and urls.py their were 15 models present in models.py so created 15 apis for both get and post method
image
image

### CODE REVIEW

**Review by piyush-451 — COMMENTED** [review]
Approved @akshatnema

**Comment by akshatnema** [comment]
@PranshuNayak @samay-rgb Kindly review this PR.

**Comment by akshatnema** [comment]
@Mayankingale123 please correct the PR title.

File: Pipfile.lock [hunk 1]
```diff
@@ -0,0 +1,1480 @@
+{
```
Comment (akshatnema) [inline Pipfile.lock]: @Mayankingale123 Kindly remove the changes of this file from this PR

Comment (Mayankingale123) [reply Pipfile.lock]: I have removed pipfile.lock

Comment (akshatnema) [reply Pipfile.lock]: Kindly update the PR title also.

Comment (Mayankingale123) [reply Pipfile.lock]: updated

**Pull request merged**